functions.py

```diff
@@ -0,0 +1,111 @@
+import time
+import turtle
+import random
+
+score = 4
+score_a = 0
+score_b = 0
+player_a = "Left Player"
+player_b = "Right Player"
+
+
+def create_game_win(left_rocket, right_rocket):
+    win = turtle.Screen()
+    win.title("Game")
+    win.bgcolor("black")
+    win.setup(width=800, height=600)
+    win.tracer(0)
+    win.listen()
+    win.onkeypress(lambda: rocket_up(left_rocket), "w")
+    win.onkeypress(lambda: rocket_down(left_rocket), "s")
+    win.onkeypress(lambda: rocket_up(right_rocket), "Up")
+    win.onkeypress(lambda: rocket_down(right_rocket), "Down")
+    win.update()
+    return win
+
+
+def creeate_roket(color, postion_cord):
+    rocket = turtle.Turtle()
+    rocket.speed(0)
+    rocket.shape("square")
+    rocket.color(color)
+    rocket.shapesize(stretch_len=1, stretch_wid=5)
+    rocket.penup()
+    rocket.goto(postion_cord, 0)
+    return rocket
+
+
+# create ball
+def create_ball():
+    ball = turtle.Turtle()
+    ball.speed(0)
+    ball.shape("turtle")
+    ball.color("yellow")
+    ball.penup()
+    ball.goto(0, 0)
+    ball.dx = random.randint(2, 10)
+    ball.dy = random.randint(2, 10)
+    return ball
+
+
+# create_score_board
+def create_score_board():
+    pen = turtle.Turtle()
+    pen.speed(0)
+    pen.shape("square")
+    pen.color("white")
+    pen.penup()
+    pen.hideturtle()
+    pen.goto(0, 260)
```

Comment:
All the coordinates of the objects depends on the game window size. Therefore, if you want to change the game window size, you need not to forget to change all the objects coordinates inside the window. This violates Dependency Inversion Principle and is a clear path to bugs.

All the objects you create inside `functions.py` are abstractions that are intended to be used by the outside code. At the same time, the window size is an implementation detail, and abstractions should be agnostic from the implementation details.

I suggest calculating the coordinates you need the objects to be placed at, depending on the field size.